package.json

```diff
@@ -75,6 +75,9 @@
         "tabtab": "2.2.2",
         "through2": "3.0.1",
         "uuid": "3.3.2",
+        "sync-request": "6.0.0",
+        "underscore.string": "3.3.4",
+        "@openapitools/openapi-generator-cli": "0.0.9-3.3.4",
```

Comment:
Please use a fixed version here as per Policy 4 https://www.jhipster.tech/policies/

Reply:
That's actually the latest stable version for the  of the `cli-3.3.4`, check the versions here https://www.npmjs.com/package/@openapitools/openapi-generator-cli/v/0.0.9-3.3.4. 

An alternative could be to use `cli-3.3.4` (as the README suggests) but I'm afraid that the `cli-` format might point to a new `0.0.10-3.3.4` in the future.

Reply:
Indeed it's the only available version tag ! I didn't know that they didn't support Semver. I have opened an issue on their tracker: https://github.com/OpenAPITools/openapi-generator-cli/issues/14